### PR TITLE
feat: added ui documentation for doc management. (PBC-123)

### PR DESCRIPTION
**Pull request by ZuhaaJamani**
Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you checked all new or changed links?

Is there anything else you would like us to know?

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

File: docs/05_components/doc-management/doc-management-ui/01_ui-configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+"@genesislcap/pbc-documents-ui": "~1"
+```
+
+:::important
+You can check the latest available version at [genesisnpm](https://www.npmjs.com/~genesisnpm).
+:::
+
+In order for the component to appear in your app, the route needs to be set up in `client/src/routes/config.ts`:
+
+```typescript
```
Comment (GuiWar) [inline docs/05_components/doc-management/doc-management-ui/01_ui-configuration.mdx:85]: Which part of this file I need to add that code?

**Comment by GuiWar** [comment]
@wjhendry can you review on that please?

**Comment by wjhendry** [comment]
> @wjhendry can you review on that please?

Yes, but later

**Comment by GuiWar** [comment]
@ZuhaaJamani , We got conflicts in this branch. Can you resolve those conflicts, and then I can merge this branch into mine, then we can proceed from there. Do you agree? Otherwise, there will be an infinite amount of conflicts, since my branch is still a work in progress.

**Pull request merged**